src/bp/lang-server/service/index.ts

```diff
@@ -19,6 +25,14 @@ const FAST_TEXT_MODEL_REGEX = /^(\w+)\.(\w+)\.(\d+)\.bin$/i
 // Examples: "scope.en.bpe.model" "bp.en.bpe.model"
 const BPE_MODEL_REGEX = /^(\w+)\.(\w+)\.bpe\.model$/i
 
+const checkEnoughRam = (langNb: number, dims: number): RamInfos => {
+  return {
+    used: Number.parseFloat((os.freemem() / (1024 * 1024 * 1024)).toFixed(2)),
```

Comment:
checkout `bytes` package

Reply:
Actually, there's a method called bytesToString which will use the correct naming